lib/modules/manager/npm/post-update/npm.ts

```diff
@@ -265,18 +265,28 @@ export function divideWorkspaceAndRootDeps(
         // stop when the first match is found and
         // add workspaceDir to workspaces set and upgrade object
         for (const workspacePattern of workspacePatterns) {
-          if (minimatch(workspacePattern).match(workspaceDir)) {
+          const massagedPattern = (workspacePattern as string).replace(
```

Comment:
isn't this already a string? 🤔

Reply:
It's `any`

Reply:
okay, we should fix the types in a separate PR